include_core/omrport.h

```diff
@@ -228,6 +228,10 @@
 #define OMRPORT_VMEM_MEMORY_MODE_SHARE_FILE_OPEN 0x000000200
 #define OMRPORT_VMEM_MEMORY_MODE_MMAP_HUGE_PAGES 0x000000400
 #define OMRPORT_VMEM_MEMORY_MODE_DOUBLE_MAP_AVAILABLE 0x000000800
+/* OMRPORT_VMEM_MEMORY_MODE_SHARE_TMP_FILE_OPEN is ignored if
+ * OMRPORT_VMEM_MEMORY_MODE_SHARE_FILE_OPEN is specified as well.
+ */
+#define OMRPORT_VMEM_MEMORY_MODE_SHARE_TMP_FILE_OPEN 0x000001000
```

Comment:
I thought about this some more: Perhaps it's better to required that `OMRPORT_VMEM_MEMORY_MODE_SHARE_FILE_OPEN` is specified if `OMRPORT_VMEM_MEMORY_MODE_SHARE_TMP_FILE_OPEN` is being requested.

Also, platforms other than Linux should not simply ignore `OMRPORT_VMEM_MEMORY_MODE_SHARE_TMP_FILE_OPEN`: they should fail because they cannot satifsy the request.

Reply:
I have updated the commit to require `OMRPORT_VMEM_MEMORY_MODE_SHARE_FILE_OPEN` if `OMRPORT_VMEM_MEMORY_MODE_SHARE_TMP_FILE_OPEN` is set.
Also I updated the other platforms to fail if `OMRPORT_VMEM_MEMORY_MODE_SHARE_TMP_FILE_OPEN` is requested.